docs/source/markdown/podman-generate-systemd.1.md

```diff
@@ -39,7 +39,7 @@ Override the default stop timeout for the container with the given value.
 **--restart-policy**=*policy*
 
 Set the systemd restart policy.  The restart-policy must be one of: "no", "on-success", "on-failure", "on-abnormal",
-"on-watchdog", "on-abort", or "always".  The default policy is *on-failure*.
+"on-watchdog", "on-abort", "unless-stopped", or "always".  The default policy is *on-failure*.
```

Comment:
alphabetize these too like prior comment.

Reply:
`podman generate systemd --restart-policy` does not support `unless-stopped`. Systemd only supports the policies as currently documented. see: `man systemd.service`

Reply:
> `podman generate systemd --restart-policy` does not support `unless-stopped`. Systemd only supports the policies as currently documented. see: `man systemd.service`

Oops, I found that out yesterday too. (https://github.com/containers/podman/pull/6522#issuecomment-699190770)
I'm wondering if this is something that PR #6522 doesn't implement and we should refine our support for this policy in the future?